### PR TITLE
Adding support to override parameters in serviceFabricDeploy Task. (#…

### PR DESCRIPTION
**Pull request by hiyadav**
…6964)

* Adding support to override parameters in serviceFabricDeploy Task.

* resolving comments and adding L0

* resolving failing unit tests.

* Changing help text for Override Application Parameters task input.

### CODE REVIEW

File: Tasks/ServiceFabricDeploy/utilities.ps1 [hunk 1]
```diff
@@ -110,6 +110,40 @@ function Read-PublishProfile
...
+function Get-OverridenApplicationParameters
+{
+    Param (
+        [String]
+        $ApplicationManifestPath
+    )
+
+    $overrideParameters = @{}
+    $applicationManifestXml = [Xml] (Get-Content -LiteralPath $ApplicationManifestPath)
+    foreach ($param in $applicationManifestXml.ApplicationManifest.Parameters.Parameter)
```
Comment (sachinma) [inline Tasks/ServiceFabricDeploy/utilities.ps1]: Can there be null ref if the xml doesn't have parameters section or zero parameters

Comment (hiyadav) [reply Tasks/ServiceFabricDeploy/utilities.ps1]: I ran some tests for this, doesn't seem to fail. Should we continue ?

Comment (sachinma) [reply Tasks/ServiceFabricDeploy/utilities.ps1]: Can you try this with a sample manifest with zero parameters and no parameters section and see this code doesn't result in failure

Comment (hiyadav) [reply Tasks/ServiceFabricDeploy/utilities.ps1]: I had tried and did not failed. Tried again now, the code change doesn't throw.

**Pull request merged**